src/lib/factories/__tests__/reporting.test.ts

```diff
@@ -0,0 +1,24 @@
+import { defaultUserMessageLead, userMessageLeadFactory } from "../reporting"
+
+describe("Reporting service factories", () => {
```

Comment:
Isn't testing the factories a bit of an overkill?

Reply:
Hmm maybe, it does not hurt I guess. (I would have actually forgotten to spread the listings object without the test...)